tests/unittests/utils/test_depth_anything.py

```
@@ -228,3 +231,139 @@ def test_none_frame_size_skips_resize(self):
+                super().__init__()
+                self.filepath = filepath
+
+        class _FakeCollection:
+            def __init__(self, samples):
+                self._samples = samples
+
+            def iter_samples(self, autosave=True, progress=None):
+                return iter(self._samples)
+
+        class _FakeFilenameMaker:
+            def __init__(self, output_dir, rel_dir=None, ignore_existing=False):
```

Ruff (0.15.5)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)